src/iproject.ts

```diff
@@ -68,39 +106,127 @@ export class IProject {
     return Uri.file(path.join(this.workspaceFolder.uri.fsPath, `.env`));
   }
 
-  public async read() {
-    const content = await workspace.fs.readFile(this.getIProjFilePath());
-    this.state = IProject.validateIProject(content.toString());
+  public async addToIncludePaths(includePath: string) {
```

Comment:
rename the parameter to `includeDir` to differentiate from the `iProject.includePath` field